### PR TITLE
add support for symfony

### PR DESCRIPTION
**Pull request by lenlorijn**
This PR would allow Symfony projects to be used under the *.symfony.dev url.
The implementation is very similar to how Magento 2 is implemented.

### CODE REVIEW

**Comment by Caroga** [comment]
Looks like a good setup, just would prefer changing $MAGE_ROOT => $SYMFONY_ROOT so it wouldn't mix up with the rest. 

+1

**Comment by Caroga** [comment]
I checked out your modifications on my laptop and got everything up and running properly. I did just now noticed the duplicated files for the server configuration inside `/sites-enabled` and `/sites-available`.
Would try to symlink it just like the rest ?

**Comment by lenlorijn** [comment]
Hey @Caroga ,
That should totally be a symlink! Don't know why it's not.
I will change it and update the pull-request.

File: nginx/nginx/sites-available/vhost/symfony [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  set $customer "default";
+  set $project "default";
+
+  if ( $host ~ "^([^\.]+)\.([^\.]+)\.symfony\.dev$"  ) {
+     set $customer "$1";
+     set $project "$2";
+  }
+
+  set $MAGE_ROOT /data/$customer/$project;
```
Comment (Caroga) [inline nginx/nginx/sites-available/vhost/symfony]: Don't forget this var :+1:

**Comment by Caroga** [comment]
Hey @lenlorijn, would you be able to update `nginx/nginx/sites-available/vhost/symfony` with the new var as well? This way @JeroenBoersma can merge it and we can all profit from your addition.

**Comment by lenlorijn** [comment]
Sorry about that. Fixed.

**Comment by JeroenBoersma** [comment]
Thanks @lenlorijn for the PR and @Caroga for the support

**Pull request merged**